src/api/custom-labels.js

```diff
@@ -0,0 +1,40 @@
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/observable/of';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/observable/throw';
+
+import {
+  parseResponseBody,
+  getHeaders,
+} from './common';
+
+const API_URL = '/eholdings/custom-labels';
+
+const createUrl = (baseUrl) => {
```

Comment:
We have a function with the same purpose in `api/agreements.js`. I would suggest to generilize it and move to `api/common`

Reply:
We can try using currying here